client/packages/common/src/hooks/useQueryParams/types.ts

```diff
@@ -13,6 +13,10 @@ export type FilterRule = {
 
 export type FilterBy = Record<string, FilterRule | null>;
 export type FilterByWithBoolean = Record<string, FilterRule | null | boolean>;
+export type FilterByWithStringAndBool = Record<
+  string,
+  FilterRule | null | boolean | string
+>;
```

Comment:
This seems a bit silly, but I'd rather not pull on this thread any further at the moment.